perf/benchmark/README.md

```diff
@@ -124,8 +124,8 @@ optional arguments:
                         none, telemetryv2
   --client CLIENT       where to run the test from
   --server SERVER       pod ip of the server
-  --perf PERF           also run perf and produce flame graph
-  --ingress INGRESS     run traffic through ingress, should be a valid URL
+  --perf PERF           run perf and produce flame graph, default is False
```

Comment:
can we rename the argument to flame_graph or sth else, the name perf is too ambiguous